pkg/tests/tasks/security/certificate/cert_manager_test.go

```diff
@@ -14,101 +15,70 @@ import (
 
 func TestCertManager(t *testing.T) {
 	test.NewTest(t).Id("T38").Groups(test.Full, test.ARM, test.InterOp).Run(func(t test.TestHelper) {
+		fooNs := "foo"
 
-		ns := "foo"
-		cert := "cert-manager"
 		t.Cleanup(func() {
-			oc.DeleteNamespace(t, cert)
-			oc.RecreateNamespace(t, ns)
+			shell.ExecuteIgnoreError(t, "helm uninstall istio-csr -n istio-system")
+			shell.ExecuteIgnoreError(t, "helm uninstall cert-manager -n cert-manager")
+			oc.DeleteNamespace(t, "cert-manager")
+			oc.RecreateNamespace(t, fooNs)
 		})
 
 		t.LogStep("Uninstall the SMCP")
-		oc.RecreateNamespace(t, meshNamespace)
+		oc.RecreateNamespace(t, "istio-system")
```

Comment:
I removed `meshNamespace`, because cert-manger objects applied from istio-operator repo have hardcoded istio-system namespace, so even `meshNamespace` shouldn't be configurable in this case.

Reply:
Hmm, this might cause issues at some point - when someone runs the test suite with a custom MESHNAMESPACE env var value. So, ideally, we should modify the templates to honor this env var instead of hardcoding the value for one test.

Reply:
Ok, I agree. I will copy those files and make namespaces configurable.

Reply:
Done